requirements.txt

```diff
@@ -10,3 +10,4 @@ six==1.10.0
 SQLAlchemy==1.1.6
 Werkzeug==0.12.1
 requests_mock==1.3.0
+pymsql=0.7.11
```

Comment:
This is only needed on the VM, right?
If so, please make a file called `requirements_prod.txt` (or make this one `requirements_dev.txt`) so we can differentiate what is needed on dev. vs. prod.